pages/api/users.ts

```diff
@@ -13,4 +13,547 @@ export default async function handler(
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+    const users = await permit.api.listUsers();
+    res.status(200).json(users);
+
+    const users = await permit.api.listUsers();
+    const throttling = users.reduce(
+        (acc, u) => ({
+            [u.id]: u.rateLimit,
+            ...acc,
+        }), {});
+    setCacheEngine(throttling)
```

Comment:
Missing ;

Reply:
It’s always :exploding_head: to see a backend developer find a missing `;` in a 500 line PR, but can’t see why a 100px button has a redundant 50px indentation hiding half of it’s text.